lib/clients/AzureClient.py

```diff
@@ -0,0 +1,459 @@
+                self.resource_group, instance_id
+            )
+            return instance.location
+        except Exception as error:
+            self.logger.error(
+                '[Azure] ERROR: Unable to determine the availability zone of instance {}.\n{}'.format(instance_id, error))
+            return None
+
+    def get_container(self):
+        try:
+            container_props = self.block_blob_service.get_container_properties(
+                self.CONTAINER)
+            # Test if the container is accessible
+            key = 'AccessTestByServiceFabrikPythonLibrary'
+            self.block_blob_service.create_blob_from_text(
+                self.CONTAINER,
+                key,
+                'This is a sample text'
+            )
+            self.block_blob_service.delete_blob(
+                self.CONTAINER,
+                key
+            )
+            return container_props
+        except Exception as error:
+            self.logger.error('[Azure] [STORAGE] ERROR: Unable to find or access container {}.\n{}'.format(
+                self.CONTAINER, error))
+            return None
+
+    def get_snapshot(self, snapshot_name):
+        try:
+            snapshot = self.compute_client.snapshots.get(
+                self.resource_group, snapshot_name)
+            return Snapshot(snapshot.name, snapshot.disk_size_gb, snapshot.provisioning_state)
+        except Exception as error:
+            self.logger.error(
+                '[Azure] ERROR: Unable to find or access snapshot {}.\n{}'.format(
+                    snapshot_name, error))
+            return None
+
+    def get_volume(self, volume_name):
+        try:
+            volume = self.compute_client.disks.get(
+                self.resource_group, volume_name)
+            return Volume(volume.name, volume.provisioning_state, volume.disk_size_gb)
+        except Exception as error:
+            self.logger.error(
+                '[Azure] ERROR: Unable to find or access volume/disk {}.\n{}'.format(
+                    volume_name, error))
+            return None
+
+    def get_attached_volumes_for_instance(self, instance_id):
+        instance = self.compute_client.virtual_machines.get(
+            self.resource_group, instance_id
+        )
+        try:
+            volumeList = []
+            for disk in instance.storage_profile.data_disks:
+                device = None
+                if disk.lun == 0:
+                    device = self.shell(
+                        'cat /proc/mounts | grep {} | grep /dev/'
+                        .format(self.DIRECTORY_DATA)).split(' ')[0][:8]
+                elif disk.lun == 1:
+                    device = self.shell(
+                        'cat /proc/mounts | grep {}'
+                        .format(self.DIRECTORY_PERSISTENT)).split(' ')[0][:8]
+
+                volumeList.append(
+                    Volume(disk.name, 'none', disk.disk_size_gb, device))
```

Comment:
what is the value 'none' here ? can you declare it as a constant ?

Reply:
Status of the volume. used only once, that's why didn't put in a constant.